scripts/wheel/build_wheel.py

```diff
@@ -9,37 +9,25 @@
 from pathlib import Path
 from string import Template
 
-
-MODULES = [
-    "mrmeshpy",
-    "mrmeshnumpy",
-    "mrviewerpy",
-]
-
-WHEEL_SCRIPT_DIR = Path(__file__).parent.resolve()
-WHEEL_ROOT_DIR = WHEEL_SCRIPT_DIR / "meshlib"
-WHEEL_SRC_DIR = WHEEL_ROOT_DIR / "meshlib"
-SOURCE_DIR = (WHEEL_SCRIPT_DIR / ".." / "..").resolve()
-
-SYSTEM = platform.system()
-LIB_EXTENSION = {
-    'Darwin': ".so",
-    'Linux': ".so",
-    'Windows': ".pyd",
-}[SYSTEM]
-LIB_DIR = {
-    'Darwin': SOURCE_DIR / "build" / "Release" / "bin" / "meshlib",
-    'Linux': SOURCE_DIR / "build" / "Release" / "bin" / "meshlib",
-    'Windows': SOURCE_DIR / "source" / "x64" / "Release",
-}[SYSTEM]
-
+import build_constans
+import create_stubs
+
+CONSTANTS = build_constans.get_build_consts()
+MODULES = CONSTANTS['MODULES']
+WHEEL_SCRIPT_DIR = CONSTANTS['WHEEL_SCRIPT_DIR']
+WHEEL_ROOT_DIR = CONSTANTS['WHEEL_ROOT_DIR']
+WHEEL_SRC_DIR = CONSTANTS['WHEEL_SRC_DIR']
+SOURCE_DIR = CONSTANTS['SOURCE_DIR']
+SYSTEM = CONSTANTS['SYSTEM']
+LIB_EXTENSION = CONSTANTS['LIB_EXTENSION']
+LIB_DIR = CONSTANTS['LIB_DIR']
```

Comment:
Can it be replaced with variable declaration in the `build_constants.py` file and using `from build_constans import *`?